src/service/auth_service.py

```diff
@@ -100,3 +103,56 @@ def logout(data):
         except Exception as error:
             current_app.logger.error(error)
             return internal_err_resp()
+
+    @staticmethod
+    def forget(email,url):
+        try:
+            # Fetch user data
+            if not (user := UserModel.query.filter_by(email=email).first()):
+                pass
+
+            expires = datetime.timedelta(hours=24)
+            reset_token = create_access_token(identity=user.uuid, expires_delta=expires)
+            
+            user.reset_password_token=reset_token
+            resp = make_response("Something went wrong while sending the password reset email",400)
+            if (mailjet.sendForget(user,url))=="error":
+                return resp
+
+            db.session.add(user)
+            db.session.commit()
+            resp = message(True, "If your account exist, you will find an email to recover your password in your mailbox")
+
+            return resp
+        except Exception as error:
+            current_app.logger.error(error)
+            return internal_err_resp()
+
+    @staticmethod
+    def reset(data,url):
+        
+        reset_token = data['reset_password_token']
+        password = data['password']
+        uuid = decode_token(reset_token)['identity']
+        print(uuid)
+        try:
+            # Fetch user data
+            if not (user := UserModel.query.filter_by(uuid=uuid).first()):
+                return err_resp(
+                    "User token not found.",
```

Comment:
```suggestion
                    "Invalid token.",
```
L'utilisateur renvoi juste le token lui, pas le user, donc le message n'est pas forcément pertinent